### PR TITLE
Pin miniforge version and update packages

### PR DESCRIPTION
**Pull request by eliotwrobson**

### CODE REVIEW

File: images/plbase/python-requirements.txt [hunk 1]
```diff
@@ -1,41 +1,41 @@
 ansi2html==1.9.2
 chevron==0.14.0
-coloraide==3.3.1
-Faker==28.1.0
+coloraide==4.0
```
Comment (eliotwrobson) [inline images/plbase/python-requirements.txt:3]: The breaking changes were pretty minimal for this, I don't think anyone is using the color science features that got messed with.

File: images/plbase/plbase-install.sh [hunk 2]
```diff
@@ -70,7 +70,7 @@ dnf -y autoremove
 echo "setting up conda..."
 cd /
 arch=`uname -m`
-curl -LO https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-${arch}.sh
+curl -LO https://github.com/conda-forge/miniforge/releases/download/24.3.0-0/Miniforge3-Linux-${arch}.sh
```
Comment (nwalters512) [inline images/plbase/plbase-install.sh:73]: Can you leave a comment here explaining why this is pinned and which version of Python we're pinning to?

File: images/plbase/python-requirements.txt (hunk 1 above)
Comment (nwalters512) [reply images/plbase/python-requirements.txt:3]: Man, Faker really loves their major versions...

**Review by nwalters512 — APPROVED** [review]
LGTM, plus or minus a comment in `plbase-install.sh`.

File: images/plbase/plbase-install.sh (hunk 2 above)
Comment (eliotwrobson) [reply images/plbase/plbase-install.sh:74]: https://github.com/PrairieLearn/PrairieLearn/pull/10769/commits/e512f97162d40608f880f4542442070a62f47955

**Pull request merged**